VerizonAds/src/main/java/com/applovin/mediation/adapters/VerizonAdsMediationAdapter.java

```diff
@@ -514,8 +465,7 @@ else if ( adFormat == MaxAdFormat.LEADER )
     //endregion
 
     //region Listeners implementation
-    private class InterstitialListener
-            implements InterstitialAdFactory.InterstitialAdFactoryListener, InterstitialAd.InterstitialAdListener
+    private class InterstitialListener implements InterstitialAd.InterstitialAdListener
```

Comment:
Could you run auto-formatting on this file? `implements` should be on a new line